tools/abel/setup.py

```diff
@@ -26,5 +26,5 @@
     install_requires=[
         'absl-py', 'pyfiglet', 'argparse', 'pyyaml', 'google-api-python-client'
     ],
-    python_requires='>=3.9',
+    python_requires='>=3.11',
```

Comment:
I propose to keep it = for now so we dont encounter future changes/surprises, wdyt?